grocerease/src/components/GroceryListItem.js

```diff
@@ -1,17 +1,52 @@
 import { useState } from 'react'
+import axios from 'axios';
 
-const GroceryListItem = ({item}) => {
+
+const GroceryListItem = ({item, token}) => {
     console.log(item)
-    const [itemCount, setItemCount] = useState(item.item_quantity);
+    const [itemData, setItemData] = useState(item);
+    const [itemCount,setItemCount] = useState();
```

Comment:
These two lines are going to cause major bugs. You can't take data from props and put them into state.